bokeh/sampledata/movies_data.py

```diff
@@ -4,7 +4,15 @@
 #
 # The full license is in the file LICENSE.txt, distributed with this software.
 #-----------------------------------------------------------------------------
-'''
+''' A small subset of data from the `Open Movie Database`_.
+
+Data is licensed `CC BY-NC 4.0`_.
+
+This modules contains an attribute ``movie_path`` that contains the path to a
+sqlite database with the data.
```

Comment:
Also a somewhat long sentence that could be split up:
```suggestion
This modules has an attribute ``movie_path``. This attribute contains the path
to a SQLite database with the data.
```

Also, I believe the preferred capitalization on is https://www.sqlite.org/src/doc/trunk/README.md is SQLite